src/player.cpp

```diff
@@ -254,10 +309,10 @@ bool Player::getPause(unsigned int handle)
     return soloud.getPause(handle);
 }
 
-
 void Player::setRelativePlaySpeed(unsigned int handle, float speed)
 {
-    if (speed < 0.05) speed = 0.05;
+    if (speed < 0.05)
```

Comment:
[nit] I think this should just throw (here or elsewhere) when `speed` becomes `0.0` or less.

Reply:
Yes, I think this could be done in `SoLoud.setRelativePlaySpeed()`, but, if I remember well, this functionality can be used with a Slider an throwing could fill the console log. So I preferred a "silent" solution. How it sounds for you? I also wrote this limit in the `SoLoud.setRelativePlaySpeed()` doc.

Reply:
Not needed to address in this PR, but generally I'd like to avoid these silent "non-fails". It makes sense for the demo, but it's weird to have this hidden "feature" than probably nobody will be aware of (since it's hidden two layers deep).

To address the issue for the Slider, just use `minValue: 0.05` there, or do this conversion in the `example/` Dart code.

Reply:
If it's possible to crash here, I'd crash. But take it as a nit, or a simple vote. If you don't like failing fast in this instance, I understand. Merge whichever way you want.